lms/djangoapps/onboarding/models.py

```diff
@@ -94,6 +109,8 @@ class TotalEmployee(models.Model):
     code = models.CharField(max_length=10, unique=True)
     label = models.CharField(max_length=256)
 
+    objects = TotalEmployeeManager()
```

Comment:
Can we make 'objects' more explanatory that says its a manager like 'total_employee_manager'.

Reply:
@muak50 actually here we are overriding default manager of Model objects.